nodejs/week1/avg.js

```diff
@@ -0,0 +1,13 @@
+function getAverage(a=0,b=0,c=0) {
```

Comment:
This restricts the function to only be able to handle maximum three values. You can just use an array as the parameter of the function, and then it can handle any amount of values.